src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -34,7 +34,7 @@
                 Trash
             </a>
         </div>
-        <div class="col-sm-10 px-0">
+        <div class="col-sm-10 px-0 ez-location-info">
```

Comment:
If we already have `px-0` here I would go and use `pb-4` instead of introducing new css class.